lib/model/model.go

```diff
@@ -2105,6 +2107,9 @@ func (m *model) SetIgnores(folder string, content []string) error {
 	if !ok {
 		return fmt.Errorf("folder %s does not exist", cfg.Description())
 	}
+	if cfg.Type == config.FolderTypeReceiveEncrypted {
+		return nil
+	}
 
```

Comment:
This seems lossy, as you can still create the file manually, so don't see much point in doing this.